Wiz/wiz-cloud-configuration-findings/ingest/parser.yml

```diff
@@ -0,0 +1,58 @@
+name: wiz-cloud-configuration-findings
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.firstSeenAt}}" # TODO: check maybe we should use `analyzedAt`
```

Comment:
I agree. As the connector uses `analyzedAt` to save its progress, `analyzedAt` seems more suitable here.
`firstSeenAt` can land into `event.start`